src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -257,13 +284,12 @@ public Task<ActionResult<ImportStatus>> GetDataUploadStatus(Guid releaseId, stri
                 .HandleFailuresOrOk();
         }
 
-        [HttpGet("release/{releaseId}/data/{fileName}/delete-plan")]
+        [HttpGet("release/{releaseId}/data/{releaseFileReferenceId}/delete-plan")]
```

Comment:
Thanks I've done this now. I used `releaseFileReferenceId` rather than `fileId` to avoid confusion between `ReleaseFileReference`  and `ReleaseFile`.

Yes it is a bad name isn't it!